test/classes/deitz/class/tree.chpl

```diff
@@ -10,14 +10,52 @@ class BinaryTree {
   }
 }
 
-iter BinaryTree.postOrder(): eltType {
+//
+// Old code uses a recursive iterator, which has trouble compiling when
+// called by writeln now that the <~> operator throws. See private issue
+// #644.
```

Comment:
It seems this should be a public issue if it's:
 * an issue a user could encounter
 * something we want to refer to in the test system

In any case, I think this is just https://github.com/chapel-lang/chapel/issues/7134

Reply:
Thanks, I had no idea about #7134.